x-pack/plugins/osquery/public/shared_components/osquery_response_action_type/index.tsx

```diff
@@ -139,13 +139,17 @@ const OsqueryResponseActionParamsFormComponent = forwardRef<
 
   useEffectOnce(() => {
     if (defaultParams && defaultParams.id) {
-      const { packId, ...restParams } = defaultParams;
+      const { packId, ecsMapping, ...restParams } = defaultParams;
       map(restParams, (value, key: keyof OsqueryResponseActionsParamsFormFields) => {
```

Comment:
```suggestion
      forEach(restParams, (value, key: keyof OsqueryResponseActionsParamsFormFields) => {
```

Reply:
forEach doesnt get type of 'key' as I want it to. Do you mind leaving it a map?
```
TS2769: No overload matches this call.   The last overload gave the following error.     Argument of type '(value: any, key: keyof OsqueryResponseActionsParamsFormFields) => void' is not assignable to parameter of type 'ObjectIterator<any, any>'.       Types of parameters 'key' and 'key' are incompatible.         Type 'string' is not assignable to type 'keyof OsqueryResponseActionsParamsFormFields'. 
```

Reply:
I'll try to type defaultParams stronger.